test/plugins/razorpay/utils/mockRazorpay.ts

```
@@ -207,12 +210,39 @@ export const createMockTransaction = (overrides: Record<string, any> = {}) => {
+        ) => Promise<unknown>)
+      | ((callback: (tx: unknown) => unknown) => unknown);
+  };
+
+  // biome-ignore lint/suspicious/noThenProperty: Intentional thenable for database mock
+  const mockDb: MockDb = {
     // The "then" method is required to make the mock object "thenable" (Promise-like),
     // allowing it to be awaited directly in the application code (e.g. await db.select()...).
     // Without this, "await" would return the object itself instead of the query result.
-    then: (resolve: any, reject: any) => mockDb.execute().then(resolve, reject),
-  };
+    then: (
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)